hadoop-tools/hadoop-distcp/src/test/java/org/apache/hadoop/tools/mapred/TestCopyCommitter.java

```diff
@@ -569,6 +572,7 @@ private void testCommitWithChecksumMismatch(boolean skipCrc)
                 fs, new Path(sourceBase + srcFilename), null,
                 fs, new Path(targetBase + srcFilename),
                 sourceCurrStatus.getLen()));
+        assertThat(log.getOutput(), containsString("Checksum not equal"));
```

Comment:
wrong import & assertThat.
This import
```
import static org.assertj.core.api.Assertions.assertThat;
```

and it works like this
```
        assertThat(log.getOutput()).contains("Checksum not equal");
```

Reply:
Copied the hamcrest version from "org.apache.hadoop.conf.TestReconfiguration". 

But changed to the Assertions.assertThat one.